### ISSUE TITLE
Append to server.properties not overwrite

### ISSUE THREAD

**Issue by justare** [bitnami/charts#1142]
Would it make sense to have the option to append to server.properties rather than overwrite with the config option.

We are using acknowledgements in Spring Boot Stream, but groups are getting rebalanced before the batch completes. So I wanted to change the max.poll.records, and i would prefer not to create config map.

We are using Helmstack on top of Helm and would also like to avoid a fork to get it working.

**Comment by tompizmor** [comment]
Can you let us know which chart are you talking about? Take into account this repository hosts 50~ charts.

**Comment by justare** [comment]
100% my bad... 

https://github.com/bitnami/charts/tree/master/bitnami/kafka

**Comment by tompizmor** [comment]
Don't worry! 

We are working on a generic way to set Kafka settings by mapping env vars to configuration values. Other users have also requested this feature in the past. 

I will update this issue once it is available.

**Comment by justare** [comment]
Thanks a bunch!
That would be really helpful, also to tweak other settings. Is there a timeframe for this? (Sorry for being pushy)

**Comment by tompizmor** [comment]
We are working on that but I am afraid that we cannot give you any specific ETA.

**Comment by justare** [comment]
Any progress?

**Comment by tompizmor** [comment]
Yes, we made progress on it but we couldn't release it yet. We plan to have it ready this week or beginning next week.

**Comment by tompizmor** [comment]
Hi @justare,

I just wanted to let you know that we have finished implementing the dynamic ENV->property parser.

Now, each env var starting by KAFKA_CFG is translated into a server.properties configuration entry.

The [configuration](https://github.com/bitnami/bitnami-docker-kafka#configuration) section in the README has been updated to document the use of this new feature.

The bitnami/kafka chart was also updated (Version 3.0.0). Check [this section](https://github.com/bitnami/charts/tree/master/bitnami/kafka#setting-custom-parameters) in the chart readme for more info
